packages/aws-cdk/toolkit/lib/cli-io-host.ts

```diff
@@ -0,0 +1,118 @@
+import * as chalk from 'chalk';
+
+/**
+ * Basic message structure for toolkit notifications.
+ * Messages are emitted by the toolkit and handled by the IoHost.
+ *
+ * @param time The time the message was emitted.
```

Comment:
i know i'm giving lots of nits here but i would expect these parameter docstrings to be on the parameters themselves, not on the interface (the way 99% of CDK code is written). is there a reason why you've written i this way? 

i suspect LLMs.

Reply:
Nope not LLM, just decided to write it that way. I'll change it.